general.el

```diff
@@ -2596,8 +2596,13 @@ or if the `:no-autoload' keyword argument is non-nil, return nil."
   ;; altered args will be passed to the autoloads and handler functions
   (defun use-package-normalize/:general (_name _keyword general-arglists)
     "Return a plist containing the original ARGLISTS and autoloadable symbols."
```

Comment:
Could you update the docstring since all it does now is return general-arglists?

Reply:
Done.